build/integration-test.sh

```diff
@@ -32,7 +32,7 @@ OC_APPS4_4="MysqlDBDepConfig4_4|MongoDBDepConfig4_4|PostgreSQLDepConfig4_4"
 OC_APPS4_5="MysqlDBDepConfig4_5|MongoDBDepConfig4_5|PostgreSQLDepConfig4_5"
```

Comment:
Should we just remove older versions? I'm confident that we don't support 3.11 anymore but we can reduce the tests to just the latest version

Reply:
From the way the code is written it looked like we wanted to support some specific legacy versions in addition to the latest. This ticket was opened to track this idea: https://github.com/kanisterio/kanister/issues/1512